src/OrchardCore.Modules/OrchardCore.Contents/Feeds/CommonFeedItemBuilder.cs

```diff
@@ -46,7 +46,10 @@ public async Task PopulateAsync(FeedContext context)
 
                     feedItem.Element.SetElementValue("title", contentItem.DisplayText);
                     feedItem.Element.Add(link);
-                    feedItem.Element.SetElementValue("description", bodyAspect.Body.ToString());
```

Comment:
This could be written like this: 
```
feedItem.Element.SetElementValue("description", bodyAspect.Body?.ToString());
```

or like this 

```
feedItem.Element.SetElementValue("description", Convert.ToString(bodyAspect.Body));
```